benchmarks/decoders/benchmark_decoders.py

```diff
@@ -197,7 +197,12 @@ def get_consecutive_frames_from_video(self, video_file, numFramesToDecode):
         return frames
 
 
-def get_video_path(filename: str) -> str:
+def get_test_resource_path(filename: str) -> str:
+    if not __package__:
```

Comment:
I don't understand why `__package__` is present here and not in fbcode. I also don't understand why test resources are available internally as `resource = importlib.resources.files(__package__).joinpath(filename)`, but not on OSS.
Before merging this PR, I think we should either: 

- understand and document what each block is doing and why it works
- replace the `if not __package__` check with a simpler `if IN_FBCODE:` block (or something similar) which would be more explicit.

Reply:
Agreed that `if IN_FBCODE` is a better check. That is what we do in a similar chunk of code for testing: [utils.py#L34](https://github.com/pytorch/torchcodec/blob/e6a4161edeed22d1157817c42bc8ceaa4f47a787/test/utils.py#L34).

My understanding of why we have different behavior depending on fbcode or not:

* In fbcode, we use `buck run //path/to/target:` to execute the benchmark. Buck creates a temporary directory with all relevant files for executing the benchmark, including all resources. Where these files actually end up is owned by buck. Hence, we have to ask `importlib` to tell use where the resources are. The code we're using as an internal best-practice.
* Externally, we just run benchmark on the command line: `python3 ./benchmarks/decoders/benchmark_decoders.py`. In that case, the file system is as it appears to be, and we can just access the files as one would expect.

Reply:
I added a function that checks for the presence of an environment variable that seems to be present running inside fbcode.